### ISSUE TITLE
Link text + URL

### ISSUE THREAD

**Issue by PelinSmines** [DPGAlliance/publicgoods-review-webapp#29]
When an applicant wants to add a link in text field, they should be able to hyperlink it to the text that they select. 

Currently all hyperlinks use the word "Link" regardless of the URL.

**Comment by skylark1848** [comment]
Hi @PelinSmines, is this the user facing submission form? Is this for all text fields in the form?

**Comment by PelinSmines** [comment]
> Hi @PelinSmines, is this the user facing submission form? Is this for all text fields in the form?

Yes @skylark1848 This is the DPG nomination form and for all the text fields in the form.
